### ISSUE TITLE
Spider-man shattered dimension almost works without issues

### ISSUE THREAD

**Issue by warriormaster12** [Joshua-Ashton/d9vk#397]
I don't if this is d9vk or wine specific issue but when I run the game with d9vk enabled and try to play Spider-man 2099 mission this happens (Image link: https://drive.google.com/file/d/1Bf3Tsxdpy3KUIZEvpupvbW0XwCHIN2vC/view?usp=sharing) and game freezes at that point which hasn't happened with running wine only. This should happen instead(https://youtu.be/tuFokJhvdAQ?t=144)

If the information wasn't helpful enough, let me know.

`Running /home/alexander/.local/share/lutris/runners/wine/lutris-4.16-x86_64/bin/wine /media/GamesAndPrograms/Games/Others/Spider-man Shattered Dimensions/Game.exe
esync: up and running.
000b:fixme:thread:create_user_shared_data_thread Creating user shared data update thread.
000b:fixme:winediag:start_process Wine TkG 4.16 is a testing version containing experimental patches.
000b:fixme:winediag:start_process Please don't report bugs about it on winehq.org and use https://github.com/Tk-Glitch/PKGBUILDS/issues instead.
000f:fixme:thread:create_user_shared_data_thread Creating user shared data update thread.
0012:fixme:thread:create_user_shared_data_thread Creating user shared data update thread.
0019:fixme:thread:create_user_shared_data_thread Creating user shared data update thread.
001b:fixme:ntdll:EtwEventRegister ({1a3c3605-aa85-4b19-aa24-bb74bc365059}, 0x180007fe4, 0x18000f120, 0x18000f720) stub.
001b:fixme:ntdll:EtwEventUnregister (deadbeef) stub.
001e:fixme:plugplay:pnp_manager_device_pnp Unhandled PnP request 0x7.
001b:fixme:ntdll:EtwEventRegister ({1a3c3605-aa85-4b19-aa24-bb74bc365059}, 0x180007fe4, 0x18000f120, 0x18000f720) stub.
001b:fixme:ntdll:EtwEventUnregister (deadbeef) stub.
0020:fixme:thread:create_user_shared_data_thread Creating user shared data update thread.
001e:fixme:ntdll:EtwEventRegister ({1a3c3605-aa85-4b19-aa24-bb74bc365059}, 0x180007fe4, 0x18000f120, 0x18000f720) stub.
001e:fixme:ntdll:EtwEventUnregister (deadbeef) stub.
0022:fixme:ntdll:EtwEventRegister ({1a3c3605-aa85-4b19-aa24-bb74bc365059}, 0x180007fe4, 0x18000f120, 0x18000f720) stub.
0022:fixme:ntdll:EtwEventUnregister (deadbeef) stub.
0028:fixme:thread:create_user_shared_data_thread Creating user shared data update thread.
000f:fixme:ntdll:EtwEventRegister ({1a3c3605-aa85-4b19-aa24-bb74bc365059}, 0x180007fe4, 0x18000f120, 0x18000f720) stub.
000f:fixme:ntdll:EtwEventUnregister (deadbeef) stub.
info:  Game: Game.exe
info:  DXVK: d9vk-0.21-lollihop
warn:  OpenVR: Failed to locate module
info:  Enabled instance extensions:
info:    VK_KHR_get_physical_device_properties2
info:    VK_KHR_surface
info:    VK_KHR_win32_surface
warn:  D3D9: VK_FORMAT_D16_UNORM_S8_UINT -> VK_FORMAT_D24_UNORM_S8_UINT
info:  GeForce GTX 1060 6GB:
info:    Driver: 435.21.0
info:    Vulkan: 1.1.109
info:    Memory Heap[0]: 
info:      Size: 6144 MiB
info:      Flags: 0x1
info:      Memory Type[7]: Property Flags = 0x1
info:      Memory Type[8]: Property Flags = 0x1
info:    Memory Heap[1]: 
info:      Size: 11961 MiB
info:      Flags: 0x0
info:      Memory Type[0]: Property Flags = 0x0
info:      Memory Type[1]: Property Flags = 0x0
info:      Memory Type[2]: Property Flags = 0x0
info:      Memory Type[3]: Property Flags = 0x0
info:      Memory Type[4]: Property Flags = 0x0
info:      Memory Type[5]: Property Flags = 0x0
info:      Memory Type[6]: Property Flags = 0x0
info:      Memory Type[9]: Property Flags = 0x6
info:      Memory Type[10]: Property Flags = 0xe
info:  Process set as DPI aware
0009:fixme:gameux:GameExplorerImpl_VerifyAccess (0016ED10, L"", 0131F8E0)
0009:fixme:system:SystemParametersInfoW Unimplemented action: 61 (SPI_SETACCESSTIMEOUT)
0009:fixme:system:SystemParametersInfoW Unimplemented action: 51 (SPI_SETFILTERKEYS)
0009:fixme:system:SystemParametersInfoW Unimplemented action: 63 (SPI_SETSERIALKEYS)
0009:fixme:system:SystemParametersInfoW Unimplemented action: 59 (SPI_SETSTICKYKEYS)
0009:fixme:system:SystemParametersInfoW Unimplemented action: 53 (SPI_SETTOGGLEKEYS)
0009:fixme:wbemprox:client_security_SetBlanket F7248C10, 00176AC0, 10, 0, (null), 3, 3, 00000000, 0x00000000
0009:fixme:wbemprox:client_security_Release F7248C10
0009:fixme:wbemprox:enum_class_object_Next timeout not supported
0009:fixme:wbemprox:client_security_SetBlanket F7248C10, 00172288, 10, 0, (null), 3, 3, 00000000, 0x00000000
0009:fixme:wbemprox:client_security_Release F7248C10
info:  Game: Game.exe
info:  DXVK: d9vk-0.21-lollihop
warn:  OpenVR: Failed to locate module
info:  Enabled instance extensions:
info:    VK_KHR_get_physical_device_properties2
info:    VK_KHR_surface
info:    VK_KHR_win32_surface
warn:  D3D9: VK_FORMAT_D16_UNORM_S8_UINT -> VK_FORMAT_D24_UNORM_S8_UINT
info:  GeForce GTX 1060 6GB:
info:    Driver: 435.21.0
info:    Vulkan: 1.1.109
info:    Memory Heap[0]: 
info:      Size: 6144 MiB
info:      Flags: 0x1
info:      Memory Type[7]: Property Flags = 0x1
info:      Memory Type[8]: Property Flags = 0x1
info:    Memory Heap[1]: 
info:      Size: 11961 MiB
info:      Flags: 0x0
info:      Memory Type[0]: Property Flags = 0x0
info:      Memory Type[1]: Property Flags = 0x0
info:      Memory Type[2]: Property Flags = 0x0
info:      Memory Type[3]: Property Flags = 0x0
info:      Memory Type[4]: Property Flags = 0x0
info:      Memory Type[5]: Property Flags = 0x0
info:      Memory Type[6]: Property Flags = 0x0
info:      Memory Type[9]: Property Flags = 0x6
info:      Memory Type[10]: Property Flags = 0xe
info:  Process set as DPI aware
info:  Game: Game.exe
info:  DXVK: d9vk-0.21-lollihop
warn:  OpenVR: Failed to locate module
info:  Enabled instance extensions:
info:    VK_KHR_get_physical_device_properties2
info:    VK_KHR_surface
info:    VK_KHR_win32_surface
warn:  D3D9: VK_FORMAT_D16_UNORM_S8_UINT -> VK_FORMAT_D24_UNORM_S8_UINT
info:  GeForce GTX 1060 6GB:
info:    Driver: 435.21.0
info:    Vulkan: 1.1.109
info:    Memory Heap[0]: 
info:      Size: 6144 MiB
info:      Flags: 0x1
info:      Memory Type[7]: Property Flags = 0x1
info:      Memory Type[8]: Property Flags = 0x1
info:    Memory Heap[1]: 
info:      Size: 11961 MiB
info:      Flags: 0x0
info:      Memory Type[0]: Property Flags = 0x0
info:      Memory Type[1]: Property Flags = 0x0
info:      Memory Type[2]: Property Flags = 0x0
info:      Memory Type[3]: Property Flags = 0x0
info:      Memory Type[4]: Property Flags = 0x0
info:      Memory Type[5]: Property Flags = 0x0
info:      Memory Type[6]: Property Flags = 0x0
info:      Memory Type[9]: Property Flags = 0x6
info:      Memory Type[10]: Property Flags = 0xe
info:  Process set as DPI aware
info:  Device properties:
info:    Device name:     : GeForce GTX 1060 6GB
info:    Driver version   : 435.21.0
info:  Enabled device extensions:
info:    VK_EXT_conditional_rendering
info:    VK_EXT_depth_clip_enable
info:    VK_EXT_host_query_reset
info:    VK_EXT_shader_viewport_index_layer
info:    VK_EXT_transform_feedback
info:    VK_EXT_vertex_attribute_divisor
info:    VK_KHR_create_renderpass2
info:    VK_KHR_dedicated_allocation
info:    VK_KHR_depth_stencil_resolve
info:    VK_KHR_descriptor_update_template
info:    VK_KHR_draw_indirect_count
info:    VK_KHR_driver_properties
info:    VK_KHR_get_memory_requirements2
info:    VK_KHR_image_format_list
info:    VK_KHR_maintenance1
info:    VK_KHR_maintenance2
info:    VK_KHR_sampler_mirror_clamp_to_edge
info:    VK_KHR_shader_draw_parameters
info:    VK_KHR_swapchain
info:  Device features:
info:    robustBufferAccess                     : 1
info:    fullDrawIndexUint32                    : 1
info:    imageCubeArray                         : 1
info:    independentBlend                       : 1
info:    geometryShader                         : 1
info:    tessellationShader                     : 0
info:    sampleRateShading                      : 1
info:    dualSrcBlend                           : 0
info:    logicOp                                : 0
info:    multiDrawIndirect                      : 0
info:    drawIndirectFirstInstance              : 0
info:    depthClamp                             : 1
info:    depthBiasClamp                         : 1
info:    fillModeNonSolid                       : 1
info:    depthBounds                            : 0
info:    multiViewport                          : 1
info:    samplerAnisotropy                      : 1
info:    textureCompressionBC                   : 1
info:    occlusionQueryPrecise                  : 1
info:    pipelineStatisticsQuery                : 1
info:    vertexPipelineStoresAndAtomics         : 1
info:    fragmentStoresAndAtomics               : 0
info:    shaderImageGatherExtended              : 0
info:    shaderStorageImageExtendedFormats      : 1
info:    shaderStorageImageReadWithoutFormat    : 0
info:    shaderStorageImageWriteWithoutFormat   : 1
info:    shaderClipDistance                     : 1
info:    shaderCullDistance                     : 1
info:    shaderFloat64                          : 0
info:    shaderInt64                            : 0
info:    variableMultisampleRate                : 0
info:  VK_EXT_conditional_rendering
info:    conditionalRendering                   : 0
info:  VK_EXT_depth_clip_enable
info:    depthClipEnable                        : 1
info:  VK_EXT_host_query_reset
info:    hostQueryReset                         : 1
info:  VK_EXT_memory_priority
info:    memoryPriority                         : 0
info:  VK_EXT_shader_demote_to_helper_invocation
info:    shaderDemoteToHelperInvocation         : 0
info:  VK_EXT_transform_feedback
info:    transformFeedback                      : 0
info:    geometryStreams                        : 0
info:  VK_EXT_vertex_attribute_divisor
info:    vertexAttributeInstanceRateDivisor     : 1
info:    vertexAttributeInstanceRateZeroDivisor : 1
info:  Queue families:
info:    Graphics : 0
info:    Transfer : 1
info:  DXVK: Read 11637 valid state cache entries
info:  DXVK: Using 2 compiler threads
info:  D3D9DeviceEx::ResetEx:
info:    Requested Presentation Parameters
info:      - Width:              1920
info:      - Height:             1080
info:      - Format:             D3D9Format::A8R8G8B8
info:      - Auto Depth Stencil: true
info:                  ^ Format: D3D9Format::D24X8
info:      - Windowed:           false
info:  Presenter: Actual swap chain properties:
info:    Format:       VK_FORMAT_B8G8R8A8_UNORM
info:    Present mode: VK_PRESENT_MODE_IMMEDIATE_KHR
info:    Buffer size:  1920x1080
info:    Image count:  2
info:  D3D9: Setting display mode: 1920x1080@0
warn:  D3D9DeviceEx::SetRenderState: Unhandled render state D3DRS_LASTPIXEL
warn:  D3D9DeviceEx::SetRenderState: Unhandled render state D3DRS_MULTISAMPLEANTIALIAS
warn:  D3D9DeviceEx::SetRenderState: Unhandled render state D3DRS_POINTSIZE_MAX
0035:fixme:msg:pack_message msg 14 (WM_ERASEBKGND) not supported yet
info:  Presenter: Actual swap chain properties:
info:    Format:       VK_FORMAT_B8G8R8A8_UNORM
info:    Present mode: VK_PRESENT_MODE_FIFO_KHR
info:    Buffer size:  1920x1080
info:    Image count:  3
0035:fixme:ddraw:ddraw7_Initialize Ignoring guid {aeb2cdd4-6e41-43ea-941c-8361cc760781}.
err:   D3D9: Reporting out of memory from tracking.
0009:fixme:dinput:IDirectInputDevice2WImpl_GetProperty Unknown property <guid-0x0017>
0009:fixme:dinput:IDirectInputDevice2WImpl_GetProperty Unknown property <guid-0x0017>
0009:fixme:dinput:IDirectInputDevice2WImpl_GetProperty Unknown property <guid-0x0017>
0009:fixme:dinput:IDirectInputDevice2WImpl_GetProperty Unknown property <guid-0x0017>
0009:fixme:dinput:IDirectInputDevice2WImpl_GetProperty Unknown property <guid-0x0017>
0009:fixme:dinput:IDirectInputDevice2WImpl_GetProperty Unknown property <guid-0x0017>
0009:fixme:dinput:IDirectInputDevice2WImpl_GetProperty Unknown property <guid-0x0017>
0009:fixme:dinput:IDirectInputDevice2WImpl_GetProperty Unknown property <guid-0x0017>
0009:fixme:dinput:IDirectInputDevice2WImpl_GetProperty Unknown property <guid-0x0017>
0009:fixme:dinput:IDirectInputDevice2WImpl_GetProperty Unknown property <guid-0x0017>
0009:fixme:dinput:IDirectInputDevice2WImpl_GetProperty Unknown property <guid-0x0017>
0009:fixme:dinput:IDirectInputDevice2WImpl_GetProperty Unknown property <guid-0x0017>
0009:fixme:dinput:IDirectInputDevice2WImpl_GetProperty Unknown property <guid-0x0017>
0009:fixme:dinput:IDirectInputDevice2WImpl_GetProperty Unknown property <guid-0x0017>
0009:fixme:dinput:IDirectInputDevice2WImpl_GetProperty Unknown property <guid-0x0017>
0009:fixme:dinput:IDirectInputDevice2WImpl_GetProperty Unknown property <guid-0x0017>
0009:fixme:dinput:IDirectInputDevice2WImpl_GetProperty Unknown property <guid-0x0017>
0009:fixme:dinput:IDirectInputDevice2WImpl_GetProperty Unknown property <guid-0x0017>
0009:fixme:dinput:IDirectInputDevice2WImpl_GetProperty Unknown property <guid-0x0017>
0009:fixme:dinput:IDirectInputDevice2WImpl_GetProperty Unknown property <guid-0x0017>
0009:fixme:dinput:IDirectInputDevice2WImpl_GetProperty Unknown property <guid-0x0017>
0009:fixme:dinput:IDirectInputDevice2WImpl_GetProperty Unknown property <guid-0x0017>
0009:fixme:dinput:IDirectInputDevice2WImpl_GetProperty Unknown property <guid-0x0017>
0009:fixme:dinput:IDirectInputDevice2WImpl_GetProperty Unknown property <guid-0x0017>
0009:fixme:dinput:IDirectInputDevice2WImpl_GetProperty Unknown property <guid-0x0017>
0009:fixme:dinput:IDirectInputDevice2WImpl_GetProperty Unknown property <guid-0x0017>
0009:fixme:dinput:IDirectInputDevice2WImpl_GetProperty Unknown property <guid-0x0017>
0009:fixme:dinput:IDirectInputDevice2WImpl_GetProperty Unknown property <guid-0x0017>
0009:fixme:dinput:IDirectInputDevice2WImpl_GetProperty Unknown property <guid-0x0017>
0009:fixme:dinput:IDirectInputDevice2WImpl_GetProperty Unknown property <guid-0x0017>
0009:fixme:dinput:IDirectInputDevice2WImpl_GetProperty Unknown property <guid-0x0017>
0009:fixme:dinput:IDirectInputDevice2WImpl_GetProperty Unknown property <guid-0x0017>
0009:fixme:dinput:IDirectInputDevice2WImpl_GetProperty Unknown property <guid-0x0017>
0009:fixme:dinput:IDirectInputDevice2WImpl_GetProperty Unknown property <guid-0x0017>
0009:fixme:dinput:IDirectInputDevice2WImpl_GetProperty Unknown property <guid-0x0017>
0009:fixme:dinput:IDirectInputDevice2WImpl_GetProperty Unknown property <guid-0x0017>
0009:fixme:dinput:IDirectInputDevice2WImpl_GetProperty Unknown property <guid-0x0017>
0009:fixme:dinput:IDirectInputDevice2WImpl_GetProperty Unknown property <guid-0x0017>
0009:fixme:dinput:IDirectInputDevice2WImpl_GetProperty Unknown property <guid-0x0017>
0009:fixme:dinput:IDirectInputDevice2WImpl_GetProperty Unknown property <guid-0x0017>
0009:fixme:dinput:IDirectInputDevice2WImpl_GetProperty Unknown property <guid-0x0017>
0009:fixme:dinput:IDirectInputDevice2WImpl_GetProperty Unknown property <guid-0x0017>
0009:fixme:dinput:IDirectInputDevice2WImpl_GetProperty Unknown property <guid-0x0017>
0009:fixme:dinput:IDirectInputDevice2WImpl_GetProperty Unknown property <guid-0x0017>
0009:fixme:dinput:IDirectInputDevice2WImpl_GetProperty Unknown property <guid-0x0017>
0009:fixme:dinput:IDirectInputDevice2WImpl_GetProperty Unknown property <guid-0x0017>
0009:fixme:dinput:IDirectInputDevice2WImpl_GetProperty Unknown property <guid-0x0017>
0009:fixme:dinput:IDirectInputDevice2WImpl_GetProperty Unknown property <guid-0x0017>
0009:fixme:dinput:IDirectInputDevice2WImpl_GetProperty Unknown property <guid-0x0017>
0009:fixme:dinput:IDirectInputDevice2WImpl_GetProperty Unknown property <guid-0x0017>
0009:fixme:dinput:IDirectInputDevice2WImpl_GetProperty Unknown property <guid-0x0017>
0009:fixme:dinput:IDirectInputDevice2WImpl_GetProperty Unknown property <guid-0x0017>
0009:fixme:dinput:IDirectInputDevice2WImpl_GetProperty Unknown property <guid-0x0017>
0009:fixme:dinput:IDirectInputDevice2WImpl_GetProperty Unknown property <guid-0x0017>
0009:fixme:dinput:IDirectInputDevice2WImpl_GetProperty Unknown property <guid-0x0017>
0009:fixme:dinput:IDirectInputDevice2WImpl_GetProperty Unknown property <guid-0x0017>
0009:fixme:dinput:IDirectInputDevice2WImpl_GetProperty Unknown property <guid-0x0017>
0009:fixme:dinput:IDirectInputDevice2WImpl_GetProperty Unknown property <guid-0x0017>
0009:fixme:dinput:IDirectInputDevice2WImpl_GetProperty Unknown property <guid-0x0017>
0009:fixme:dinput:IDirectInputDevice2WImpl_GetProperty Unknown property <guid-0x0017>
0009:fixme:dinput:IDirectInputDevice2WImpl_GetProperty Unknown property <guid-0x0017>
0009:fixme:dinput:IDirectInputDevice2WImpl_GetProperty Unknown property <guid-0x0017>
0009:fixme:dinput:IDirectInputDevice2WImpl_GetProperty Unknown property <guid-0x0017>
0009:fixme:dinput:IDirectInputDevice2WImpl_GetProperty Unknown property <guid-0x0017>
0009:fixme:dinput:IDirectInputDevice2WImpl_GetProperty Unknown property <guid-0x0017>
0009:fixme:dinput:IDirectInputDevice2WImpl_GetProperty Unknown property <guid-0x0017>
0009:fixme:dinput:IDirectInputDevice2WImpl_GetProperty Unknown property <guid-0x0017>
0009:fixme:dinput:IDirectInputDevice2WImpl_GetProperty Unknown property <guid-0x0017>
0009:fixme:dinput:IDirectInputDevice2WImpl_GetProperty Unknown property <guid-0x0017>
0009:fixme:dinput:IDirectInputDevice2WImpl_GetProperty Unknown property <guid-0x0017>
0009:fixme:dinput:IDirectInputDevice2WImpl_GetProperty Unknown property <guid-0x0017>
0009:fixme:xinput:XInputGetKeystroke (index 0, reserved 0, keystroke 0131FE08) Stub!
0009:fixme:dinput:IDirectInputDevice2WImpl_GetProperty Unknown property <guid-0x0017>
0009:fixme:dinput:IDirectInputDevice2WImpl_GetProperty Unknown property <guid-0x0017>
0009:fixme:dinput:IDirectInputDevice2WImpl_GetProperty Unknown property <guid-0x0017>
0009:fixme:dinput:IDirectInputDevice2WImpl_GetProperty Unknown property <guid-0x0017>
0009:fixme:dinput:IDirectInputDevice2WImpl_GetProperty Unknown property <guid-0x0017>
0009:fixme:dinput:IDirectInputDevice2WImpl_GetProperty Unknown property <guid-0x0017>
0009:fixme:dinput:IDirectInputDevice2WImpl_GetProperty Unknown property <guid-0x0017>
0009:fixme:dinput:IDirectInputDevice2WImpl_GetProperty Unknown property <guid-0x0017>
0009:fixme:dinput:IDirectInputDevice2WImpl_GetProperty Unknown property <guid-0x0017>
0009:fixme:dinput:IDirectInputDevice2WImpl_GetProperty Unknown property <guid-0x0017>
0009:fixme:dinput:IDirectInputDevice2WImpl_GetProperty Unknown property <guid-0x0017>
0009:fixme:dinput:IDirectInputDevice2WImpl_GetProperty Unknown property <guid-0x0017>
0009:fixme:dinput:IDirectInputDevice2WImpl_GetProperty Unknown property <guid-0x0017>
0009:fixme:dinput:IDirectInputDevice2WImpl_GetProperty Unknown property <guid-0x0017>
0009:fixme:dinput:IDirectInputDevice2WImpl_GetProperty Unknown property <guid-0x0017>
0009:fixme:dinput:IDirectInputDevice2WImpl_GetProperty Unknown property <guid-0x0017>
0009:fixme:dinput:IDirectInputDevice2WImpl_GetProperty Unknown property <guid-0x0017>
0009:fixme:dinput:IDirectInputDevice2WImpl_GetProperty Unknown property <guid-0x0017>
0009:fixme:dinput:IDirectInputDevice2WImpl_GetProperty Unknown property <guid-0x0017>
0009:fixme:dinput:IDirectInputDevice2WImpl_GetProperty Unknown property <guid-0x0017>
0009:fixme:dinput:IDirectInputDevice2WImpl_GetProperty Unknown property <guid-0x0017>
0009:fixme:dinput:IDirectInputDevice2WImpl_GetProperty Unknown property <guid-0x0017>
0009:fixme:dinput:IDirectInputDevice2WImpl_GetProperty Unknown property <guid-0x0017>
0009:fixme:dinput:IDirectInputDevice2WImpl_GetProperty Unknown property <guid-0x0017>
0009:fixme:dinput:IDirectInputDevice2WImpl_GetProperty Unknown property <guid-0x0017>
0009:fixme:dinput:IDirectInputDevice2WImpl_GetProperty Unknown property <guid-0x0017>
0009:fixme:dinput:IDirectInputDevice2WImpl_GetProperty Unknown property <guid-0x0017>
0009:fixme:dinput:IDirectInputDevice2WImpl_GetProperty Unknown property <guid-0x0017>
0009:fixme:dinput:IDirectInputDevice2WImpl_GetProperty Unknown property <guid-0x0017>
0009:fixme:dinput:IDirectInputDevice2WImpl_GetProperty Unknown property <guid-0x0017>
0009:fixme:dinput:IDirectInputDevice2WImpl_GetProperty Unknown property <guid-0x0017>
0009:fixme:dinput:IDirectInputDevice2WImpl_GetProperty Unknown property <guid-0x0017>
0009:fixme:dinput:IDirectInputDevice2WImpl_GetProperty Unknown property <guid-0x0017>
0009:fixme:dinput:IDirectInputDevice2WImpl_GetProperty Unknown property <guid-0x0017>
0009:fixme:dinput:IDirectInputDevice2WImpl_GetProperty Unknown property <guid-0x0017>
0009:fixme:dinput:IDirectInputDevice2WImpl_GetProperty Unknown property <guid-0x0017>
0009:fixme:dinput:IDirectInputDevice2WImpl_GetProperty Unknown property <guid-0x0017>
0009:fixme:dinput:IDirectInputDevice2WImpl_GetProperty Unknown property <guid-0x0017>
0009:fixme:dinput:IDirectInputDevice2WImpl_GetProperty Unknown property <guid-0x0017>
0009:fixme:dinput:IDirectInputDevice2WImpl_GetProperty Unknown property <guid-0x0017>
0009:fixme:dinput:IDirectInputDevice2WImpl_GetProperty Unknown property <guid-0x0017>
0009:fixme:dinput:IDirectInputDevice2WImpl_GetProperty Unknown property <guid-0x0017>
0009:fixme:dinput:IDirectInputDevice2WImpl_GetProperty Unknown property <guid-0x0017>
0009:fixme:dinput:IDirectInputDevice2WImpl_GetProperty Unknown property <guid-0x0017>
0009:fixme:dinput:IDirectInputDevice2WImpl_GetProperty Unknown property <guid-0x0017>
0009:fixme:dinput:IDirectInputDevice2WImpl_GetProperty Unknown property <guid-0x0017>
0009:fixme:dinput:IDirectInputDevice2WImpl_GetProperty Unknown property <guid-0x0017>
0009:fixme:dinput:IDirectInputDevice2WImpl_GetProperty Unknown property <guid-0x0017>
0009:fixme:dinput:IDirectInputDevice2WImpl_GetProperty Unknown property <guid-0x0017>
INFO: Assertion failed: 0 && "FFmpeg codec not supported!"
INFO: Assertion failed: (voice->src.ffmpeg != NULL && pBufferWMA != NULL) || (voice->src.ffmpeg == NULL && pBufferWMA == NULL)
INFO: Assertion failed: 0 && "Unsupported format tag!"
wine: Unhandled page fault on read access to 0x00000000 at address 00000000 (thread 003e), starting debugger...
005d:fixme:thread:create_user_shared_data_thread Creating user shared data update thread.
005a:fixme:dbghelp:elf_search_auxv can't find symbol in module
005a:fixme:dbghelp:elf_search_auxv can't find symbol in module
005a:fixme:dbghelp:elf_search_auxv can't find symbol in module
005a:fixme:dbghelp:elf_search_auxv can't find symbol in module
005a:fixme:dbghelp:elf_search_auxv can't find symbol in module
005a:fixme:dbghelp:elf_search_auxv can't find symbol in module
005a:fixme:dbghelp:elf_search_auxv can't find symbol in module
005a:fixme:dbghelp:elf_search_auxv can't find symbol in module
005a:fixme:dbghelp:elf_search_auxv can't find symbol in module
005a:fixme:dbghelp:elf_search_auxv can't find symbol in module
005a:fixme:dbghelp:elf_search_auxv can't find symbol in module
005a:fixme:dbghelp:elf_search_auxv can't find symbol in module
005a:fixme:dbghelp:elf_search_auxv can't find symbol in module
005a:fixme:dbghelp:elf_search_auxv can't find symbol in module
005a:fixme:dbghelp:elf_search_auxv can't find symbol in module
005a:fixme:dbghelp:elf_search_auxv can't find symbol in module
005a:fixme:dbghelp:elf_search_auxv can't find symbol in module
005a:fixme:dbghelp:elf_search_auxv can't find symbol in module
005a:fixme:dbghelp:elf_search_auxv can't find symbol in module
005a:fixme:dbghelp:elf_search_auxv can't find symbol in module
005a:fixme:dbghelp:elf_search_auxv can't find symbol in module
005a:fixme:dbghelp:elf_search_auxv can't find symbol in module
005a:fixme:dbghelp:elf_search_auxv can't find symbol in module
005a:fixme:dbghelp:elf_search_auxv can't find symbol in module
005a:fixme:dbghelp:elf_search_auxv can't find symbol in module
005a:fixme:dbghelp:elf_search_auxv can't find symbol in module
005a:fixme:dbghelp:elf_search_auxv can't find symbol in module
005a:fixme:dbghelp:elf_search_auxv can't find symbol in module
005a:fixme:dbghelp:elf_search_auxv can't find symbol in module
005a:fixme:dbghelp:elf_search_auxv can't find symbol in module
005a:fixme:dbghelp:elf_search_auxv can't find symbol in module
005a:fixme:dbghelp:elf_search_auxv can't find symbol in module
005a:fixme:dbghelp:elf_search_auxv can't find symbol in module
005a:fixme:dbghelp:elf_search_auxv can't find symbol in module
005a:fixme:dbghelp:elf_search_auxv can't find symbol in module
005a:fixme:dbghelp:elf_search_auxv can't find symbol in module
005a:fixme:dbghelp:elf_search_auxv can't find symbol in module
005a:fixme:dbghelp:elf_search_auxv can't find symbol in module
005a:fixme:dbghelp:elf_search_auxv can't find symbol in module
005a:fixme:dbghelp:elf_search_auxv can't find symbol in module
005a:fixme:dbghelp:elf_search_auxv can't find symbol in module
005a:fixme:dbghelp:elf_search_auxv can't find symbol in module
005a:fixme:dbghelp:elf_search_auxv can't find symbol in module
005a:fixme:dbghelp:elf_search_auxv can't find symbol in module
005a:fixme:dbghelp:elf_search_auxv can't find symbol in module
005a:fixme:dbghelp:elf_search_auxv can't find symbol in module
005a:fixme:dbghelp:elf_search_auxv can't find symbol in module
0035:fixme:msg:pack_message msg 14 (WM_ERASEBKGND) not supported yet
Unhandled exception: page fault on read access to 0x00000000 in 32-bit code (0x00000000).
005a:fixme:dbghelp:elf_search_auxv can't find symbol in module
Register dump:
 CS:0023 SS:002b DS:002b ES:002b FS:0063 GS:006b
 EIP:00000000 ESP:0a7dfe4c EBP:0bb6db18 EFLAGS:00010202(  R- --  I   - - - )
 EAX:0ba21938 EBX:00000000 ECX:00000000 EDX:0017efe8
 ESI:00000200 EDI:0ba16c68
Stack dump:
0x0a7dfe4c:  f00f7152 0bb6db18 0ba16c68 0ba21938
0x0a7dfe5c:  00000200 00000000 00000000 f0113ff4
0x0a7dfe6c:  00000200 00000200 00000000 00000000
0x0a7dfe7c:  00000000 f0101f9c 0017efe8 f0101c18
0x0a7dfe8c:  0bb781b0 f010082f 00000000 00000000
0x0a7dfe9c:  00000000 f0101f80 f0102000 7ce501f0
Backtrace:
=>0 0x00000000 (0x0bb6db18)
  1 0x00000000 (0x0017efe8)
0x00000000: -- no code accessible --
Modules:
Module	Address			Debug info	Name (171 modules)
PE	  360000-  394000	Deferred        rld
PE	  400000-  e16000	Deferred        game
PE	 a0d0000- a2d7000	Deferred        d3dcompiler_43
PE	 a7f0000- aeb5000	Deferred        gamelogic
PE	10000000-101ff000	Deferred        d3dx9_43
PE	62440000-626d9000	Deferred        d3d9
ELF	7a800000-7a949000	Deferred        opengl32<elf>
  \-PE	7a840000-7a949000	\               opengl32
ELF	7b400000-7b830000	Deferred        kernel32<elf>
  \-PE	7b420000-7b830000	\               kernel32
ELF	7bc00000-7bd8f000	Deferred        ntdll<elf>
  \-PE	7bc30000-7bd8f000	\               ntdll
ELF	7c000000-7c004000	Deferred        <wine-loader>
ELF	7cd82000-7cdce000	Deferred        uxtheme<elf>
  \-PE	7cd90000-7cdce000	\               uxtheme
ELF	7cdce000-7ce2c000	Deferred        libvulkan.so.1
ELF	7ce2e000-7ce35000	Deferred        libxfixes.so.3
ELF	7ce35000-7ce41000	Deferred        libxcursor.so.1
ELF	7d00b000-7d03d000	Deferred        libexpat.so.1
ELF	7d03d000-7d088000	Deferred        libfontconfig.so.1
ELF	7d088000-7d0a7000	Deferred        libz.so.1
ELF	7d0a7000-7d0e1000	Deferred        libpng16.so.16
ELF	7d0e1000-7d19e000	Deferred        libfreetype.so.6
ELF	7d19e000-7d1b1000	Deferred        libxi.so.6
ELF	7d1b1000-7d1b5000	Deferred        libxcomposite.so.1
ELF	7d1b5000-7d1c2000	Deferred        libxrandr.so.2
ELF	7d1c2000-7d1ce000	Deferred        libxrender.so.1
ELF	7d1ce000-7d1d5000	Deferred        libxxf86vm.so.1
ELF	7d1d5000-7d1d9000	Deferred        libxinerama.so.1
ELF	7d1d9000-7d265000	Deferred        setupapi<elf>
  \-PE	7d1f0000-7d265000	\               setupapi
ELF	7d265000-7d280000	Deferred        libbsd.so.0
ELF	7d280000-7d287000	Deferred        libxdmcp.so.6
ELF	7d287000-7d2b3000	Deferred        libxcb.so.1
ELF	7d2b3000-7d3fd000	Deferred        libx11.so.6
ELF	7d3fd000-7d412000	Deferred        libxext.so.6
ELF	7d412000-7d4d1000	Deferred        winex11<elf>
  \-PE	7d430000-7d4d1000	\               winex11
ELF	7d4d1000-7d4ff000	Deferred        iphlpapi<elf>
  \-PE	7d4e0000-7d4ff000	\               iphlpapi
ELF	7d4ff000-7d63f000	Deferred        oleaut32<elf>
  \-PE	7d530000-7d63f000	\               oleaut32
ELF	7d63f000-7d65b000	Deferred        aclui<elf>
  \-PE	7d640000-7d65b000	\               aclui
ELF	7d65b000-7d681000	Deferred        shcore<elf>
  \-PE	7d660000-7d681000	\               shcore
ELF	7d681000-7d6e9000	Deferred        shlwapi<elf>
  \-PE	7d690000-7d6e9000	\               shlwapi
ELF	7d6e9000-7e11e000	Deferred        shell32<elf>
  \-PE	7d720000-7e11e000	\               shell32
ELF	7e11e000-7e15a000	Deferred        ws2_32<elf>
  \-PE	7e130000-7e15a000	\               ws2_32
ELF	7e15a000-7e1f1000	Deferred        rpcrt4<elf>
  \-PE	7e180000-7e1f1000	\               rpcrt4
ELF	7e1f1000-7e35f000	Deferred        ole32<elf>
  \-PE	7e220000-7e35f000	\               ole32
ELF	7e35f000-7e384000	Deferred        imm32<elf>
  \-PE	7e370000-7e384000	\               imm32
ELF	7e384000-7e3d1000	Deferred        usp10<elf>
  \-PE	7e390000-7e3d1000	\               usp10
ELF	7e3d1000-7e52f000	Deferred        comctl32<elf>
  \-PE	7e3f0000-7e52f000	\               comctl32
ELF	7e52f000-7e594000	Deferred        dinput8<elf>
  \-PE	7e540000-7e594000	\               dinput8
ELF	7e594000-7e5ab000	Deferred        version<elf>
  \-PE	7e5a0000-7e5ab000	\               version
ELF	7e5ab000-7e62f000	Deferred        advapi32<elf>
  \-PE	7e5c0000-7e62f000	\               advapi32
ELF	7e62f000-7e78a000	Deferred        gdi32<elf>
  \-PE	7e650000-7e78a000	\               gdi32
ELF	7e78a000-7e9c4000	Deferred        user32<elf>
  \-PE	7e7b0000-7e9c4000	\               user32
ELF	7e9c4000-7ea13000	Deferred        winevulkan<elf>
  \-PE	7e9e0000-7ea13000	\               winevulkan
ELF	7ea13000-7eaea000	Deferred        msvcrt<elf>
  \-PE	7ea40000-7eaea000	\               msvcrt
ELF	7eaea000-7eb02000	Deferred        vulkan-1<elf>
  \-PE	7eaf0000-7eb02000	\               vulkan-1
ELF	7ee34000-7ee48000	Deferred        wow64cpu<elf>
  \-PE	7ee40000-7ee48000	\               wow64cpu
ELF	7ee48000-7eeee000	Deferred        kernelbase<elf>
  \-PE	7ee60000-7eeee000	\               kernelbase
ELF	7eeee000-7ef02000	Deferred        libnss_files.so.2
ELF	7ef02000-7ef1d000	Deferred        libnsl.so.1
ELF	7ef1d000-7ef2b000	Deferred        libnss_nis.so.2
ELF	7ef2b000-7ef35000	Deferred        libnss_compat.so.2
ELF	7ef35000-7f000000	Deferred        libm.so.6
ELF	e9f4c000-e9fd8000	Deferred        libvorbisenc.so.2
ELF	e9fd8000-ea067000	Deferred        libsndfile.so.1
ELF	ea067000-ea200000	Deferred        libavcodec.so.58
ELF	ee773000-ee800000	Deferred        libpulsecommon-11.1.so
ELF	ef710000-ef76f000	Deferred        libflac.so.8
ELF	ef97b000-ef9d5000	Deferred        libpulse.so.0
ELF	ef9d5000-efb00000	Deferred        libsdl2-2.0.so.0
ELF	efc08000-efc33000	Deferred        libvorbis.so.0
ELF	efc33000-efccd000	Deferred        libavutil.so.56
ELF	f00d2000-f0118000	Deferred        libfaudio.so.0
ELF	f0184000-f01a2000	Deferred        libgcc_s.so.1
ELF	f01a2000-f01c7000	Deferred        xaudio2_7<elf>
  \-PE	f01b0000-f01c7000	\               xaudio2_7
ELF	f02fe000-f035d000	Deferred        libgldispatch.so.0
ELF	f035d000-f0380000	Deferred        libglx.so.0
ELF	f0380000-f03e1000	Deferred        libgl.so.1
ELF	f03e1000-f045f000	Deferred        ddraw<elf>
  \-PE	f03f0000-f045f000	\               ddraw
ELF	f0756000-f0773000	Deferred        libxcb-glx.so.0
ELF	f2986000-f455a000	Deferred        libnvidia-glcore.so.435.21
ELF	f4905000-f490d000	Deferred        libxcb-sync.so.1
ELF	f4f05000-f4f09000	Deferred        libxcb-present.so.0
ELF	f51dd000-f51e6000	Deferred        libogg.so.0
ELF	f5402000-f5407000	Deferred        libxcb-dri3.so.0
ELF	f5507000-f550a000	Deferred        libx11-xcb.so.1
ELF	f5582000-f559c000	Deferred        hid<elf>
  \-PE	f5590000-f559c000	\               hid
ELF	f5816000-f581d000	Deferred        libasyncns.so.0
ELF	f581d000-f5828000	Deferred        libwrap.so.0
ELF	f5828000-f582e000	Deferred        libpulse-simple.so.0
ELF	f586e000-f65ab000	Deferred        libnvidia-glvkspirv.so.435.21
ELF	f6857000-f6946000	Deferred        libglx_nvidia.so.0
ELF	f6b57000-f6b6b000	Deferred        libdrm.so.2
ELF	f6b6b000-f6b70000	Deferred        libnvidia-tls.so.435.21
ELF	f6b71000-f6b89000	Deferred        libgpg-error.so.0
ELF	f6b89000-f6c6a000	Deferred        libgcrypt.so.20
ELF	f6c6a000-f6c80000	Deferred        liblz4.so.1
ELF	f6c80000-f6cac000	Deferred        liblzma.so.5
ELF	f6cac000-f6d3c000	Deferred        libsystemd.so.0
ELF	f6d3c000-f6d54000	Deferred        libresolv.so.2
ELF	f6d54000-f6d59000	Deferred        libkeyutils.so.1
ELF	f6d59000-f6db4000	Deferred        libdbus-1.so.3
ELF	f6db4000-f6dc1000	Deferred        libkrb5support.so.0
ELF	f6dc1000-f6dc6000	Deferred        libcom_err.so.2
ELF	f6dc6000-f6dfb000	Deferred        libk5crypto.so.3
ELF	f6dfb000-f6ed7000	Deferred        libkrb5.so.3
ELF	f6ed7000-f6eeb000	Deferred        libavahi-client.so.3
ELF	f6eeb000-f6ef9000	Deferred        libavahi-common.so.3
ELF	f6ef9000-f6f4d000	Deferred        libgssapi_krb5.so.2
ELF	f6f4d000-f6fe0000	Deferred        libcups.so.2
ELF	f6fe0000-f7179000	Deferred        wined3d<elf>
  \-PE	f7020000-f7179000	\               wined3d
ELF	f7179000-f71ad000	Deferred        dxgi<elf>
  \-PE	f7180000-f71ad000	\               dxgi
ELF	f71ad000-f7204000	Deferred        winspool<elf>
  \-PE	f71c0000-f7204000	\               winspool
ELF	f7204000-f7249000	Deferred        wbemprox<elf>
  \-PE	f7210000-f7249000	\               wbemprox
ELF	f7249000-f7261000	Deferred        xinput1_3<elf>
  \-PE	f7250000-f7261000	\               xinput1_3
ELF	f7261000-f7289000	Deferred        gameux<elf>
  \-PE	f7270000-f7289000	\               gameux
ELF	f7306000-f730f000	Deferred        libffi.so.6
ELF	f730f000-f739a000	Deferred        libgmp.so.10
ELF	f739a000-f73d0000	Deferred        libhogweed.so.4
ELF	f73d0000-f740c000	Deferred        libnettle.so.6
ELF	f740c000-f7421000	Deferred        libtasn1.so.6
ELF	f7421000-f75a2000	Deferred        libunistring.so.2
ELF	f75a2000-f75c0000	Deferred        libidn2.so.0
ELF	f75c0000-f770e000	Deferred        libp11-kit.so.0
ELF	f770e000-f78a4000	Deferred        libgnutls.so.30
ELF	f78a4000-f799d000	Deferred        crypt32<elf>
  \-PE	f78c0000-f799d000	\               crypt32
ELF	f799d000-f79c0000	Deferred        bcrypt<elf>
  \-PE	f79a0000-f79c0000	\               bcrypt
ELF	f79c0000-f7a00000	Deferred        rsaenh<elf>
  \-PE	f79d0000-f7a00000	\               rsaenh
ELF	f7a00000-f7a04000	Deferred        libxau.so.6
ELF	f7a05000-f7a0f000	Deferred        librt.so.1
ELF	f7b11000-f7b16000	Deferred        libdl.so.2
ELF	f7b16000-f7cef000	Deferred        libc.so.6
ELF	f7cef000-f7d0e000	Deferred        libpthread.so.0
ELF	f7d0e000-f7ed7000	Dwarf           libwine.so.1
ELF	f7ed9000-f7f01000	Deferred        ld-linux.so.2
Threads:
process  tid      prio (all id:s are in hex)
00000008 (D) Z:\media\GamesAndPrograms\Games\Others\Spider-man Shattered Dimensions\Game.exe
	["Z:\media\GamesAndPrograms\Games\Others\Spider-man Shattered Dimensions\Game.exe"]
	0000003f    0
	0000003e    0 <==
	0000003d    0
	0000003c   -1
	0000003b    0
	0000003a    0
	00000039    0
	00000038    0
	00000037   -2
	00000036   -2
	00000035    0
	00000034    0
	00000033    0
	00000032   -1
	00000031    0
	00000030    0
	0000002f    0
	0000002e    0
	0000002d    0
	0000002c    0
	00000009    0
0000000e services.exe
	00000025    0
	00000021    0
	0000001d    0
	0000001a    0
	00000013    0
	00000010    0
	0000000f    0
00000011 plugplay.exe
	00000017    0
	00000016    0
	00000012    0
00000018 winedevice.exe
	00000024    0
	0000001e    0
	0000001c    0
	0000001b    0
	00000019    0
0000001f winedevice.exe
	00000026    0
	00000023    0
	00000022    0
	00000020    0
00000027 explorer.exe
	0000002b    0
	0000002a    0
	00000029    0
	00000028    0
0000005c explorer.exe
	00000060    0
	0000005f    0
	0000005e    0
	0000005d    0
System information:
    Wine build: wine-4.16.r0.gc323ef47 ( TkG Staging Esync Fsync )
    Platform: i386 (WOW64)
    Version: Windows 7
    Host system: Linux
    Host version: 5.3.1-050301-generic
Waiting on children
All children gone
Exit with returncode 5`

**Comment by Joshua-Ashton** [comment]
Hi, I am not able to debug this without an apitrace.

**Comment by warriormaster12** [comment]
@Joshua-Ashton managed to create apitrace and here's also the d9vk.log file

link to apitrace:https://drive.google.com/file/d/1YAKkZySIQtUOSzb3MGRp9OUkZTnuwXWx/view?usp=sharing

d9vk.log file:`info:  Game: Game.exe
info:  DXVK: d9vk-0.21-lollihop
warn:  OpenVR: Failed to locate module
info:  Enabled instance extensions:
info:    VK_KHR_get_physical_device_properties2
info:    VK_KHR_surface
info:    VK_KHR_win32_surface
warn:  D3D9: VK_FORMAT_D16_UNORM_S8_UINT -> VK_FORMAT_D24_UNORM_S8_UINT
info:  GeForce GTX 1060 6GB:
info:    Driver: 435.21.0
info:    Vulkan: 1.1.109
info:    Memory Heap[0]: 
info:      Size: 6144 MiB
info:      Flags: 0x1
info:      Memory Type[7]: Property Flags = 0x1
info:      Memory Type[8]: Property Flags = 0x1
info:    Memory Heap[1]: 
info:      Size: 11961 MiB
info:      Flags: 0x0
info:      Memory Type[0]: Property Flags = 0x0
info:      Memory Type[1]: Property Flags = 0x0
info:      Memory Type[2]: Property Flags = 0x0
info:      Memory Type[3]: Property Flags = 0x0
info:      Memory Type[4]: Property Flags = 0x0
info:      Memory Type[5]: Property Flags = 0x0
info:      Memory Type[6]: Property Flags = 0x0
info:      Memory Type[9]: Property Flags = 0x6
info:      Memory Type[10]: Property Flags = 0xe
info:  Process set as DPI aware
info:  Game: Game.exe
info:  DXVK: d9vk-0.21-lollihop
warn:  OpenVR: Failed to locate module
info:  Enabled instance extensions:
info:    VK_KHR_get_physical_device_properties2
info:    VK_KHR_surface
info:    VK_KHR_win32_surface
warn:  D3D9: VK_FORMAT_D16_UNORM_S8_UINT -> VK_FORMAT_D24_UNORM_S8_UINT
info:  GeForce GTX 1060 6GB:
info:    Driver: 435.21.0
info:    Vulkan: 1.1.109
info:    Memory Heap[0]: 
info:      Size: 6144 MiB
info:      Flags: 0x1
info:      Memory Type[7]: Property Flags = 0x1
info:      Memory Type[8]: Property Flags = 0x1
info:    Memory Heap[1]: 
info:      Size: 11961 MiB
info:      Flags: 0x0
info:      Memory Type[0]: Property Flags = 0x0
info:      Memory Type[1]: Property Flags = 0x0
info:      Memory Type[2]: Property Flags = 0x0
info:      Memory Type[3]: Property Flags = 0x0
info:      Memory Type[4]: Property Flags = 0x0
info:      Memory Type[5]: Property Flags = 0x0
info:      Memory Type[6]: Property Flags = 0x0
info:      Memory Type[9]: Property Flags = 0x6
info:      Memory Type[10]: Property Flags = 0xe
info:  Process set as DPI aware
info:  Game: Game.exe
info:  DXVK: d9vk-0.21-lollihop
warn:  OpenVR: Failed to locate module
info:  Enabled instance extensions:
info:    VK_KHR_get_physical_device_properties2
info:    VK_KHR_surface
info:    VK_KHR_win32_surface
warn:  D3D9: VK_FORMAT_D16_UNORM_S8_UINT -> VK_FORMAT_D24_UNORM_S8_UINT
info:  GeForce GTX 1060 6GB:
info:    Driver: 435.21.0
info:    Vulkan: 1.1.109
info:    Memory Heap[0]: 
info:      Size: 6144 MiB
info:      Flags: 0x1
info:      Memory Type[7]: Property Flags = 0x1
info:      Memory Type[8]: Property Flags = 0x1
info:    Memory Heap[1]: 
info:      Size: 11961 MiB
info:      Flags: 0x0
info:      Memory Type[0]: Property Flags = 0x0
info:      Memory Type[1]: Property Flags = 0x0
info:      Memory Type[2]: Property Flags = 0x0
info:      Memory Type[3]: Property Flags = 0x0
info:      Memory Type[4]: Property Flags = 0x0
info:      Memory Type[5]: Property Flags = 0x0
info:      Memory Type[6]: Property Flags = 0x0
info:      Memory Type[9]: Property Flags = 0x6
info:      Memory Type[10]: Property Flags = 0xe
info:  Process set as DPI aware
info:  Device properties:
info:    Device name:     : GeForce GTX 1060 6GB
info:    Driver version   : 435.21.0
info:  Enabled device extensions:
info:    VK_EXT_conditional_rendering
info:    VK_EXT_depth_clip_enable
info:    VK_EXT_host_query_reset
info:    VK_EXT_shader_viewport_index_layer
info:    VK_EXT_transform_feedback
info:    VK_EXT_vertex_attribute_divisor
info:    VK_KHR_create_renderpass2
info:    VK_KHR_dedicated_allocation
info:    VK_KHR_depth_stencil_resolve
info:    VK_KHR_descriptor_update_template
info:    VK_KHR_draw_indirect_count
info:    VK_KHR_driver_properties
info:    VK_KHR_get_memory_requirements2
info:    VK_KHR_image_format_list
info:    VK_KHR_maintenance1
info:    VK_KHR_maintenance2
info:    VK_KHR_sampler_mirror_clamp_to_edge
info:    VK_KHR_shader_draw_parameters
info:    VK_KHR_swapchain
info:  Device features:
info:    robustBufferAccess                     : 1
info:    fullDrawIndexUint32                    : 1
info:    imageCubeArray                         : 1
info:    independentBlend                       : 1
info:    geometryShader                         : 1
info:    tessellationShader                     : 0
info:    sampleRateShading                      : 1
info:    dualSrcBlend                           : 0
info:    logicOp                                : 0
info:    multiDrawIndirect                      : 0
info:    drawIndirectFirstInstance              : 0
info:    depthClamp                             : 1
info:    depthBiasClamp                         : 1
info:    fillModeNonSolid                       : 1
info:    depthBounds                            : 0
info:    multiViewport                          : 1
info:    samplerAnisotropy                      : 1
info:    textureCompressionBC                   : 1
info:    occlusionQueryPrecise                  : 1
info:    pipelineStatisticsQuery                : 1
info:    vertexPipelineStoresAndAtomics         : 1
info:    fragmentStoresAndAtomics               : 0
info:    shaderImageGatherExtended              : 0
info:    shaderStorageImageExtendedFormats      : 1
info:    shaderStorageImageReadWithoutFormat    : 0
info:    shaderStorageImageWriteWithoutFormat   : 1
info:    shaderClipDistance                     : 1
info:    shaderCullDistance                     : 1
info:    shaderFloat64                          : 0
info:    shaderInt64                            : 0
info:    variableMultisampleRate                : 0
info:  VK_EXT_conditional_rendering
info:    conditionalRendering                   : 0
info:  VK_EXT_depth_clip_enable
info:    depthClipEnable                        : 1
info:  VK_EXT_host_query_reset
info:    hostQueryReset                         : 1
info:  VK_EXT_memory_priority
info:    memoryPriority                         : 0
info:  VK_EXT_shader_demote_to_helper_invocation
info:    shaderDemoteToHelperInvocation         : 0
info:  VK_EXT_transform_feedback
info:    transformFeedback                      : 0
info:    geometryStreams                        : 0
info:  VK_EXT_vertex_attribute_divisor
info:    vertexAttributeInstanceRateDivisor     : 1
info:    vertexAttributeInstanceRateZeroDivisor : 1
info:  Queue families:
info:    Graphics : 0
info:    Transfer : 1
info:  DXVK: Read 11637 valid state cache entries
info:  DXVK: Using 2 compiler threads
info:  D3D9DeviceEx::ResetEx:
info:    Requested Presentation Parameters
info:      - Width:              1920
info:      - Height:             1080
info:      - Format:             D3D9Format::A8R8G8B8
info:      - Auto Depth Stencil: true
info:                  ^ Format: D3D9Format::D24X8
info:      - Windowed:           false
info:  Presenter: Actual swap chain properties:
info:    Format:       VK_FORMAT_B8G8R8A8_UNORM
info:    Present mode: VK_PRESENT_MODE_IMMEDIATE_KHR
info:    Buffer size:  1920x1080
info:    Image count:  2
info:  D3D9: Setting display mode: 1920x1080@0
warn:  D3D9DeviceEx::SetRenderState: Unhandled render state D3DRS_LASTPIXEL
warn:  D3D9DeviceEx::SetRenderState: Unhandled render state D3DRS_MULTISAMPLEANTIALIAS
warn:  D3D9DeviceEx::SetRenderState: Unhandled render state D3DRS_POINTSIZE_MAX
info:  Presenter: Actual swap chain properties:
info:    Format:       VK_FORMAT_B8G8R8A8_UNORM
info:    Present mode: VK_PRESENT_MODE_FIFO_KHR
info:    Buffer size:  1920x1080
info:    Image count:  3
err:   D3D9: Reporting out of memory from tracking.`

**Comment by Joshua-Ashton** [comment]
Primarily for my sanity, can you upload log files as files in future 🐸

**Comment by warriormaster12** [comment]
@Joshua-Ashton No problem :)

**Comment by warriormaster12** [comment]
The game still has the same issue with latest d9vk 0.40.1. This time though when the bug triggers, the game crashes.
